common/app/controllers/EmailSignupController.scala

```diff
@@ -113,8 +116,15 @@ class EmailSignupController(
       Action { implicit request =>
         val identityNewsletter = emailEmbedAgent.getNewsletterByName(listName)
         identityNewsletter match {
-          case Right(Some(_)) =>
-            Cached(1.day)(RevalidatableResult.Ok(views.html.emailFragmentFooter(emailLandingPage, listName)))
+          case Right(Some(newsletter)) =>
+            if (EmailSignupRecaptcha.isSwitchedOn && newsletter.signupPage.isDefined) {
+              Cached(1.day)(
```

Comment:
Previously we cached the result for 1 day. Just want to double-check there isn't a reason for extending/compressing the cache time in this scenario?

Reply:
Hmm, interesting! I hadn't thought about this. Now that I do think about it, I have some questions (not necessarily for you, but I need to find the answers nonetheless!)

* can we invalidate it in any way, like if we want to deactivate the feature switch?
* is there any reason we'd want it more/less than 1day